charts/xinference/values.yaml

```diff
@@ -62,6 +62,7 @@ serviceWorker:
   type: ClusterIP
 xinferenceSupervisor:
   supervisor:
+    image: ""
```

Comment:
Is this change still required?

Reply:
I think main `values.yaml` should include all options user can set, don't you think?

Reply:
Sure, thanks.